modules/apps/callbacks/forwarding_test.go

```diff
@@ -0,0 +1,182 @@
+package ibccallbacks_test
```

Comment:
I had to add a new file here in the callbacks module because it's the only module in which we can check the callbacks counter. This required setting up a new, small test suite that creates 3 chains.